### PR TITLE
[Fix] Readme

### PR DESCRIPTION
**Pull request by Sqyphen**
**Description**
Updates the README with recent launch and troubleshooting notes

**Implementation**
Updates some gem packages and adds notes to the README file

**Screenshot**
n/a

**.env changes**
n/a

**Additional requirements**
May need to run `bundle install` again to get latest packages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,53 @@
-# Notes on the Jeykll-ifying of labs.ft.com
+# ftlabs.github.io
+
+Github hosted blog of FT Labs
```
Comment (Lily2point0) [inline README.md]: I think GitHub, prefer a capitalised H :p

Comment (Sqyphen) [reply README.md]: 😱

Comment (Sqyphen) [reply README.md]: Done :)

File: Gemfile.lock [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
   remote: https://rubygems.org/
   specs:
-    addressable (2.5.2)
+    addressable (2.6.0)
       public_suffix (>= 2.0.2, < 4.0)
     colorator (1.1.0)
-    ffi (1.9.25)
+    ffi (1.10.0)
     forwardable-extended (2.6.0)
     jekyll (3.4.3)
```
Comment (Lily2point0) [inline Gemfile.lock]: Can you upgrade jekyll to 3.6.3 or higher, to get rid of the vulnerability?

Comment (Sqyphen) [reply Gemfile.lock]: Sure! I was going to create a separate PR for that but can do it here

Comment (Sqyphen) [reply Gemfile.lock]: _Bundler attempted to update jekyll but its version stayed the same_

ffs

Comment (Sqyphen) [reply Gemfile.lock]: Done

**Review by Lily2point0 — APPROVED** [review]

**Pull request merged**